vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/DateTimePickerAutoOpenIT.java

```diff
@@ -0,0 +1,27 @@
+package com.vaadin.flow.component.datetimepicker;
+
+import com.vaadin.flow.component.datetimepicker.testbench.DateTimePickerElement;
+import com.vaadin.flow.testutil.AbstractComponentIT;
```

Comment:
`import com.vaadin.flow.testutil.AbstractComponentIT;` should be changed to `com.vaadin.tests.AbstractComponentIT` in new ITs